production/dpl-workflow.sh

```diff
@@ -219,6 +219,14 @@ get_N() # USAGE: get_N [processor-name] [DETECTOR_NAME] [RAW|CTF|REST] [optional
   local NAME_DEFAULT="N_$4"
   echo $1:${!NAME_PROC:-$((${!NAME_FACTOR} * ${!NAME_DET:-1} * ${!NAME_DEFAULT:-1}))}
 }
+
+get_F() # get factor for arbitrary <NAME> <DEFAULT_FACTOR=1>
+{
+  local defVal=1
+  (( $# == 2 )) && defVal=${2}
+  [ -n "${!1}" ] && echo "${!1}" || echo "$defVal"
+}
+
```

Comment:
This function seems not needed, since you solved the defaults differently below. If it is not needed I would remove it, to keep the script clean and short.

Reply:
right, after adding `get_F` I've realized that passing factors directly is more compact.